README.md

```diff
@@ -1,24 +1,95 @@
-# CAR FOR YOU Example
+# CAR FOR YOU assets
 
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 ## Usage
 ```
-npm install @carforyou/example
+npm install @carforyou/assets-pkg
+```
+
+### clean `dist`
+```
+npx cfy-assets clean
+```
+
+### optimize svg from `src` to `dist`
+```
+npx cfy-assets optimize
+```
+
+### generate components from optimized svg
+```
+npx cfy-assets components
+```
+
+
+### create index
+```
+npx cfy-assets index
```

Comment:
shall we group this with the components? or will we ever want to generate them separately?

Reply:
Listings has a npm command in the package.json calling it separately. That is why I included it here as well. If you think it's not needed, I'm happy to remove it

Reply:
Further feedback appreciated if we should only expose `build` to the consumer

Reply:
I never used them standalone since we added the asset handling, so I am fine with only exposing the aggregated command. (It might also be that they were just broke up for the purpose of avoiding super long commands in the package.json)

Reply:
I'm fine with only exposing `build` and `clean`. The cases when you want to run some commands in separation are very rare. Let's revisit if they pop up often